src/main/java/org/apache/sling/maven/bundlesupport/BundleInstallMojo.java

```diff
@@ -65,7 +65,7 @@ protected File getBundleFileName() {
                 getLog().debug("No bundle found in secondary artifact " + file);
             }
         }
-        return null;
+        throw new MojoExecutionException("No attached bundle found for this Maven project, for standalone usage use goal 'install-file' instead!");
```

Comment:
Not sure if we need to more more lenient for backwards-compatibility reasons. OTOH calling this goal should either succeed or fail the build in case the bundle cannot be installed for whatever reason (this is crucial particularly for non-interactive e.g. CI builds where no one is checking the logs). Just silently (i.e. with just some log message) skipping is bad behaviour for a Maven plugin.

Reply:
with this explicit error message it's fine in my pov.